### PR TITLE
Testing To Do list Add and Remove function

### PR DESCRIPTION
**Pull request by developertariq**
- Created a new __test__ folder
- Created addRemove file
- Added scripts to test the add and remove function
- Created .babelrc file in the root directory and added some script tags in it
- Update package.json file

### CODE REVIEW

File: src/__test__/addRemove.test.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+    expect(taskFromListEl).toEqual(mockData[0]);
+    expect(taskFromLocalStorage).toEqual(mockData[0]);
+
+    localStorage.clear();
+  });
+});
+
+describe('Add method', () => {
+  test('Remove method should add task to dom and local storage', () => {
```
Comment (GzimAsani) [inline src/__test__/addRemove.test.js]: - [ ] Kindly update wording here, this is a remove method and this should remove tasks from dom and local storage

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @developertariq  and @Progress2002                                :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with your tests, they are passing :+1: 

 ## Required Changes 
- [ ] Check the comment below :arrow_double_down: 




 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/__test__/addRemove.test.js (hunk 1 above)
Comment (Progress2002) [reply src/__test__/addRemove.test.js]: @GzimAsani  Thanks for the observation. Changes have been made.

Comment (peterrobert) [reply src/__test__/addRemove.test.js]: This change has been done

**Review by peterrobert — APPROVED** [review]
Hi @developertariq, 


### Highlights!

[ ] Good feature branch name ✔️
[ ] Descriptive pull request ✔️

### OPTIONAL

- [ ] It will be great to remove the Microverse logo from your readme file. You can either replace it with your own logo or remove it completely. This will make your project even more professional.


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**